app/models/manageiq/providers/amazon/inventory/collector.rb

```diff
@@ -99,4 +99,13 @@ def health_check_members(load_balancer_name)
   def health_check_members_refs(load_balancer_name)
     @health_check_members_refs.try(:[], load_balancer_name) || []
   end
+
+  def max_filter_size
+    200
```

Comment:
I assume this isn't some constant in the aws-sdk gem?

Reply:
haven't found that, that would be nice :-)